### ISSUE TITLE
Compilation Error : digitalPintoInterrupt

### ISSUE THREAD

**Issue by rkuo2000** [ambiot/ambpro2_arduino#165]
WARNING: library MPU6050 claims to run on avr, samd, sam, esp8266, esp32, stm32, renesas, renesas_portenta, renesas_uno architecture(s) and may be incompatible with your current board which runs on AmebaPro2 architecture(s).
D:\Users\rkuo2\Documents\Arduino\examples\IMU\MPU6050_DMP6_Teapot\MPU6050_DMP6_Teapot.ino: In function 'void setup()':
D:\Users\rkuo2\Documents\Arduino\examples\IMU\MPU6050_DMP6_Teapot\MPU6050_DMP6_Teapot.ino:135:18: error: 'digitalPinToInterrupt' was not declared in this scope; did you mean 'digitalPinToPort'?
  135 |     Serial.print(digitalPinToInterrupt(INTERRUPT_PIN));
      |                  ^~~~~~~~~~~~~~~~~~~~~
      |                  digitalPinToPort
D:\Users\rkuo2\Documents\Arduino\examples\IMU\MPU6050_DMP6_Teapot\MPU6050_DMP6_Teapot.ino:137:5: error: 'attachInterrupt' was not declared in this scope
  137 |     attachInterrupt(digitalPinToInterrupt(INTERRUPT_PIN), dmpDataReady, RISING);
      |     ^~~~~~~~~~~~~~~

exit status 1

Compilation error: 'digitalPinToInterrupt' was not declared in this scope; did you mean 'digitalPinToPort'?

**Comment by M-ichae-l** [comment]
`WARNING: library MPU6050 claims to run on avr, samd, sam, esp8266, esp32, stm32, renesas, renesas_portenta, renesas_uno architecture(s) and may be incompatible with your current board which runs on AmebaPro2 architecture(s).`

I believe this is the reason.

**Comment by rkuo2000** [comment]
AMB82-MINI seems not support AttachInterrupt(digitalPinToInterrupt) in Arduino

**Comment by M-ichae-l** [comment]
MPU6050 is supported by latest early release version 4.0.7

**Comment by rkuo2000** [comment]
Yes, but digitalPintoInterrupt is not supported, so need to comment it out if used.

**Comment by M-ichae-l** [comment]
#368 the PR is added the Arduino external interrupt set for SDK. [API documents](https://docs.arduino.cc/language-reference/en/functions/external-interrupts/attachInterrupt/)
